components/event-bus/test/integration/publish_push_test.go

```diff
@@ -279,3 +279,68 @@ func Test_pushRequestShouldNotIncludeKymaTopicHeader(t *testing.T) {
 		t.Fatalf("request to endpoint should not include the header %s", headerKymaTopic)
 	}
 }
+
+func Test_sameSubjectSubscribersInDifferentNamespacesShouldReceiveEventsOfThatSubject(t *testing.T) {
+	// create two subscriptions with the same name in two different namespaces to the same subject
+	name := "test-subscription"
+	subscription1 := util.NewSubscription(name, "namespace-1", subscriberServerV1.URL+util.SubServer1EventsPath, eventType, eventTypeVersion, sourceIDV1)
+	subscription2 := util.NewSubscription(name, "namespace-2", subscriberServerV2.URL+util.SubServer2EventsPath, eventType, eventTypeVersion, sourceIDV1)
+
+	// handle the two subscriptions
+	pushApplication := initPushApplication()
```

Comment:
All the initialization should be done in the `TestMain` including starting the applications.

Please also fix `Test_pushRequestShouldNotIncludeKymaTopicHeader` which has similiar issues